Realm/RLMValueBase.h

```diff
@@ -0,0 +1,35 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class RLMObjectBase, RLMProperty;
+@protocol RLMValue;
+
+@interface RLMValueBase : NSObject
```

Comment:
This definitely needs comments containing an explanation of why this exists, as it's not readily clear nor is it _really_ a base class for `RLMValue`.

Reply:
This is a left over from the `AnyRealmValue` class, I'll clean it up